sqs/main.tf

```diff
@@ -64,43 +64,35 @@ resource "aws_sqs_queue" "dlq_with_sse" {
   sqs_managed_sse_enabled   = true
 }
 
-resource "aws_cloudwatch_metric_alarm" "dlq_metric_alarm" {
-  alarm_name          = "${var.queue_name}-messages-visible--dlq-alarm"
+
+resource "aws_cloudwatch_metric_alarm" "dlq_metric_messages_visible_alarm" {
+  alarm_name          = "${var.queue_name}" - dlq-messages-visible-alarm
```

Comment:
I think this needs to be in the format `"${var.queue_name}-dlq-messages-visible-alarm" as we don't want spaces in the alarm names. There's a few places this needs changing in this PR.